server/rpc.go

```diff
@@ -34,6 +34,7 @@ import (
 
 const HTTPTimeout = 8 * time.Second
 const GRPCTimeout = 8 * time.Second
+const GRPCConnectTimeout = 3 * time.Second
```

Comment:
Can we reduce `GRPCTimeout` to 3 seconds and reuse that? Would https://github.com/livepeer/go-livepeer/issues/517 help? 8 seconds is a long time to wait for a job confirmation.

Reply:
I'm not sure. Timeout was raised to 8 second because of slow `orchestrator.GetJob` call.
Internally it calls 
`	job, err := orch.node.Eth.GetJob(big.NewInt(jid))`
that returns some job, but it doesn't have assigned transcoder, so it calls
`		ta, err := orch.node.Eth.AssignedTranscoder(job)`
and that call fails. If job doesn't have assigned transcoder in the blockchain, why it would have it in db? Or I'm missing something?

Reply:
> If job doesn't have assigned transcoder in the blockchain, why it would have it in db? Or I'm missing something?

This probably depends on what the root of the issue is. 

If the issue is with the RPC call failing (regardless of the geth node's current block, eg light client) then we'd be likely to see this error sporadically, even if the GetJob call for the same job has succeeded in the past. Looking up the job in the DB is more likely to help here. This seems to be the more likely scenario; pinging @yondonfu to confirm.

If the root is the geth blockchain being out-of-sync from the broadcaster/Infura, the DB is less likely to help. We'd be better served by sending the CreationBlock along with the job.